datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -25,7 +26,10 @@ def ensure_unicode(s):
     return s
 
 
-to_string = ensure_unicode if PY3 else ensure_bytes
+if TYPE_CHECKING:
```

Comment:
what's this for?

Reply:
The `to_string` implementation depends on runtime data (`PY3`) that Mypy can't know about. The end result on py2 or py3 is the same: `to_string()` turns stuff into what the language considers as strings, i.e. `str`.

So here I tell Mypy "hey `to_string()` is really just a synonym for `str` in all cases", so that it can properly interpret stuff like `value = to_string(something)` (otherwise `value` would be inferred as `Any` instead of `str`).

Reply:
Also a note on `TYPE_CHECKING`: it's basically `False` at runtime, but type checkers treat it as `True` when parsing the source code.